packages/ui/packages/IntegerInput/src/index.js

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { englishNumber } from '@snappmarket/helpers';
+import Input from '../../Input';
+
+const isValid = (value, min, max) =>
+  value !== '-' &&
+  (min === undefined || value >= min) &&
+  (max === undefined || value <= max);
+
+const IntegerInput = ({
+  value,
+  min,
+  max,
+  onChange,
+  minMessage,
+  maxMessage,
+  nanMessage,
+  placeholder,
+  type,
```

Comment:
This is better to handle with a prop on `Input` component, this could increase size without any strong reason